### ISSUE TITLE
banner image accessibility

### ISSUE THREAD

**Issue by banksashleym** [JacksonCollege/jc-theme-2015#182]
The banner images for each site section is also giving an accessibility error. When I run the [WAVE](http://wave.webaim.org/report#/https://www.jccmi.edu/financial-aid/) scanner on an inside page, it's saying that the banner image is missing alt text, even though there is alt text on the actual image.

**Comment by banksashleym** [comment]
Hi Joel, there seems to be an issue now with all child page banners? They aren't showing up anymore? Thanks!

**Comment by joelworsham** [comment]
Ah shoot yes. My apologies! All set.

**Comment by banksashleym** [comment]
Thanks so much!
